hpe3parclient/client.py

```diff
@@ -4286,7 +4286,6 @@ def admitRemoteCopyLinks(
             cmd = ['admitrcopylink', targetName, source_target_port_pair]
             response = self._run(cmd)
             if response != []:
-                response = response[0]
```

Comment:
Initially we were returning only part of the message, now it will return whole message that
we are getting from storeserv